plone/app/contenttypes/behaviors/viewlets.py

```diff
@@ -1,10 +1,10 @@
-from plone.app.contenttypes.behaviors.leadimage import ILeadImage
+from plone.app.contenttypes.behaviors.leadimage import ILeadImageBehavior
 from plone.app.layout.viewlets import ViewletBase
 
 
 class LeadImageViewlet(ViewletBase):
     """A simple viewlet which renders leadimage"""
 
     def update(self):
-        self.context = ILeadImage(self.context)
-        self.available = True if self.context.image else False
```

Comment:
@thet Are you sure this viewlet doesn't use a template that depends on having `self.context` set to the behavior adapter?

Reply:
This change was just due to a test failure where the viewlet wasn't rendered anymore.
But I'll double check it!

Reply:
Seems not a problem.